jsx/Form.js

```diff
@@ -1525,6 +1525,37 @@ ButtonElement.defaultProps = {
   },
 };
 
+/**
+  * Call To Action (CTA) component
+  * React wrapper for <button> element that is used for Call to Actions, usually
+  * outside the context of forms.
+  */
+ class CTA extends Component {
```

Comment:
is there any plan to deprecate the ButtonElement component in favor of this?

Reply:
No, although the Button Element may end up using the CTA component. I'm not sure if this is the right file for the CTA, but we need some sort of button that doesn't conform to the `Form` formatting so that it can be used in table cells, above datatables, etc.